features/command_line_interface/show_progress.feature

```diff
@@ -15,8 +15,8 @@ Feature: Show progress
       .S
 
       mixed_files/dirty.rb -- 2 warnings:
-        [4]:UncommunicativeMethodName: Smelly#x has the name 'x' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Method-Name.md]
```

Comment:
Instead of changing this everywhere, what do you think about rather adding "--no-documentation" to reek calls in our cukes and then having a separate new feature in features/command_line_interface that is there to make sure that our documentation links don't break?

Reply:
Done. I made the documentation flag have no effect on JSON and YAML reports, because it makes sense and to match the documentation as per `reek -h`.